Uni_life_tracker/app/src/main/java/com/generic/ult/SignUpActivity.java

```diff
@@ -96,9 +112,63 @@ public void signupInput(View v) {
                 validate(textInputEmail) | validate(textInputPassword))) {
             // Here we can get all the info we need
             // For example to get the email you can do textInputEmail.getEditTest().getText().toString()
+
         }
 
 
     }
 
+    private void saveName() {
+        String name = Objects.requireNonNull(textInputName.getEditText()).getText().toString();
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(this);
+        @SuppressLint("CommitPrefEdits") SharedPreferences.Editor prefsEditor = prefs.edit();
+        prefsEditor.putString("NAME", name);
+        prefsEditor.apply();
+
+    }
+    private void saveUni() {
+        String uni = Objects.requireNonNull(textInputUni.getEditText()).getText().toString();
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(this);
+        @SuppressLint("CommitPrefEdits") SharedPreferences.Editor prefsEditor = prefs.edit();
+        prefsEditor.putString("UNIVERSITY", uni);
+        prefsEditor.apply();
+
+    }
+    private void saveEmail() {
+        String email = Objects.requireNonNull(textInputEmail.getEditText()).getText().toString();
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(this);
+        @SuppressLint("CommitPrefEdits") SharedPreferences.Editor prefsEditor = prefs.edit();
+        prefsEditor.putString("EMAIL", email);
+        prefsEditor.apply();
+
+    }
+    private void savePassword() {
+        String password = Objects.requireNonNull(textInputPassword.getEditText()).getText().toString();
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(this);
+        @SuppressLint("CommitPrefEdits") SharedPreferences.Editor prefsEditor = prefs.edit();
+        prefsEditor.putString("PASSWORD", password);
+        prefsEditor.apply();
+
+    }
+
+    private String getName() {
```

Comment:
I was wondering if these get statements are needed because it doesn't look like its being used

Reply:
For now don't think so, but maybe later if we choose to somehow need the user's name in the UI, maybe to display it as a welcome?